src/widgets/alignment.zig

```diff
@@ -10,3 +10,47 @@ pub fn center(parent: Window, cols: u16, rows: u16) Window {
         .height = rows,
     });
 }
+
+pub fn topLeft(parent: Window, cols: u16, rows: u16) Window {
+    const y_off: u16 = 0;
+    const x_off: u16 = 0;
+    return parent.child(.{
+        .x_off = x_off,
+        .y_off = y_off,
+        .width = .{ .limit = cols },
+        .height = .{ .limit = rows }
```

Comment:
I updated the `child` api the other day. `width` and `height` are now a `?u16`. All of these calls can be:

```zig
.width = cols,
.height = rows,
```

Reply:
I see! Updated now.